services/ui-src/src/components/report/ReportPageWrapper.tsx

```diff
@@ -103,15 +104,30 @@ export const ReportPageWrapper = () => {
                     Previous
                   </Button>
                 )}
-                {parentPage.index < parentPage.childPageIds.length - 1 && (
+                {parentPage.index < parentPage.childPageIds.length - 1 &&
+                  parentPage.index !== 0 && (
+                    <Button
+                      onClick={() => SetPageIndex(parentPage.index + 1)}
+                      marginLeft="auto"
+                    >
+                      Continue
+                    </Button>
+                  )}
+                {parentPage.index == 0 && (
                   <Button
                     onClick={() => SetPageIndex(parentPage.index + 1)}
                     marginLeft="auto"
+                    justifyContent={"flex-end"}
                   >
                     Continue
                   </Button>
                 )}
               </Flex>
+              <Box flex="auto" width="97.5%">
+                {parentPage.index == 0 && (
+                  <Page elements={[praStatement] ?? []}></Page>
```

Comment:
This feels hacky and I'm open to feedback if there's a better way to pull the pra disclosure statment?

Reply:
It shouldn't need to be its own page, just an element right? Unless we're doing something where we support many elements, but it looks like it is just the PRA element.

Reply:
This also may render the pra statement on nested sets of pages, I believe this is the index within the parent's children

Reply:
Correct. No need to pull it in via the `Page`. Pulling it in with the `paragraphElement` function but not sure about `index` and `formKey` props. Since design confirmed that we can assume other reports will also have this paragraph, are we good to keep it here?

Reply:
`formKey` should is used for filling out answers, it might be fine to make an optional prop or exclude from a paragraph, or we make some other element that renders a block of text to the screen. Why not just make a PRADisclosure element and drop that here instead?

Reply:
** PRADisclosure** element felt the most straightforward for now 😺